### PR TITLE
Updated the troubleshooting guide according to Guy's comments

### PR DESCRIPTION
**Pull request by gal-revach**

### CODE REVIEW

File: docs/admin/troubleshooting/cluster-health-check.md [hunk 1]
```diff
@@ -27,33 +27,37 @@ Use the following steps to troubleshoot the issue:
 
     `kubectl get pods -n runai | grep -E 'runai-agent|cluster-sync|assets-sync'`
 
-2. Verify Run:ai services logs. Inspecting the logs of the Run:ai services that communicate with the CP is an essential first step to identify any error messages or connection issues.
+   If one or more of the services are not running, use the guidelines described under [Cluster has Service issues](#cluster-has-service-issues).
```
Comment (jasonnovichRunAI) [inline docs/admin/troubleshooting/cluster-health-check.md:30]: @gal-revach 
```suggestion
   If one or more of the services are not running, see [Cluster has Service issues](#cluster-has-service-issues).
```

File: docs/admin/troubleshooting/cluster-health-check.md [hunk 2]
```diff
@@ -27,33 +27,37 @@ Use the following steps to troubleshoot the issue:
...
+
+    ```bash
+    kubectl logs deployment/runai-agent -n runai
+    kubectl logs deployment/cluster-sync -n runai
+    kubectl logs deployment/assets-sync -n runai
+    ```
+
+    From the logs, try to identify the problem. If you cannot resolve the issue, continue to the next step. 
+
+4. If the issue persists after completing the previous steps, contact Run:ai support for assistance.
```
Comment (jasonnovichRunAI) [inline docs/admin/troubleshooting/cluster-health-check.md:60]: @gal-revach 
```suggestion
4. If the issue persists, contact Run:ai support for assistance.
```

File: docs/admin/troubleshooting/cluster-health-check.md [hunk 3]
```diff
@@ -27,33 +27,37 @@ Use the following steps to troubleshoot the issue:
...
+
+    Run the following command on each one of the services:
+
+    ```bash
+    kubectl logs deployment/runai-agent -n runai
+    kubectl logs deployment/cluster-sync -n runai
+    kubectl logs deployment/assets-sync -n runai
+    ```
+
+    From the logs, try to identify the problem. If you cannot resolve the issue, continue to the next step.
```
Comment (jasonnovichRunAI) [inline docs/admin/troubleshooting/cluster-health-check.md:58]: @gal-revach 
```suggestion
   Try to identify the problem from the logs. If you cannot resolve the issue, continue to the next step. 
```

File: docs/admin/troubleshooting/cluster-health-check.md (hunk 1 above)
Comment (gal-revach) [reply docs/admin/troubleshooting/cluster-health-check.md:30]: @jasonnovichRunAI looks good

**Pull request merged**